src/KafkaFlow.Retry.SqlServer/Repositories/RetryQueueItemMessageRepository.cs

```diff
@@ -8,15 +8,15 @@
 
     internal sealed class RetryQueueItemMessageRepository : IRetryQueueItemMessageRepository
     {
-        public async Task AddAsync(IDbConnection dbConnection, RetryQueueItemMessageDbo retryQueueItemMessageDbo)
+        public async Task AddAsync(IDbConnection dbConnection, RetryQueueItemMessageDbo retryQueueItemMessageDbo, string schema)
         {
             Guard.Argument(dbConnection, nameof(dbConnection)).NotNull();
             Guard.Argument(retryQueueItemMessageDbo, nameof(retryQueueItemMessageDbo)).NotNull();
 
             using (var command = dbConnection.CreateCommand())
             {
                 command.CommandType = System.Data.CommandType.Text;
-                command.CommandText = @"INSERT INTO [ItemMessages]
+                command.CommandText = $@"INSERT INTO [{schema}].[ItemMessages]
                                             (IdRetryQueueItem, [Key], Value, TopicName, Partition, Offset, UtcTimeStamp)
                                         VALUES
                                             (@idRetryQueueItem, @key, @value, @topicName, @partition, @offSet, @utcTimeStamp)";
```

Comment:
Github doesn't allow comments there, look up lines 53 and 57, you need to consider the schema also there.